CATALOG.md

```diff
@@ -1346,6 +1346,22 @@ Tags|common,operator
 |Non-Telco|Mandatory|
 |Telco|Mandatory|
 
+#### operator-valid-single-namespaced-operator-installation-tenant-namespace
```

Comment:
why not rename this test case to operator-only-single-namespace-mode-allowed-in-tenant-namespaces

Reply:
Thanks for the suggestion.